### DISCUSSION TITLE
Extension works for email (IMAP)?

### DISCUSSION THREAD

**Discussion by wslybkkr** [eblasoft/ebla-oauth#8] [Q&A]
Hello,

Can you tell if this extension also works to pull incoming emails through imap? 
So we (espocrm users) are able to use this plugin instead of buying the outlook extension from espocrm.com for only the OAuth function?

If not, are you planning to build this in the future?

**Comment by eymen-elkum** [comment]
Hello,

No, this feature only implemented on Outlook and not planned to implemented here, please note that this repository is going to be archived because Espocrm addedd it to its core one month after we released this module 

Best regards
Eymen
